HHS/Head-Start-TTADP
# update node

## Description of change

This updates node to 18.18.2 as our current version is no longer supported.

This also updates Trusworks to latest.

**Note:** We attempted to update to upgrade to a higher version but it required us to regen our lock file (and potentially cause unexpected issues).

## How to test

- Review the code.
- Make sure all tests pass.
- Perform some operations in the hub and make sure Truss components work as they did before (the drop down select in particular).


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-0


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
